backend/src/packages/subscription/subscription.service.ts

```diff
@@ -15,6 +19,10 @@ class SubscriptionService
 
   public constructor(subscriptionRepository: SubscriptionRepository) {
     this.subscriptionRepository = subscriptionRepository;
+
```

Comment:
add method a `initCron` here, then add new method a `initCrons` in the `ServerApplication` and run it in the `init` method